app/__init__.py

```diff
@@ -130,6 +133,13 @@ async def register(
     return Response(registration, status_code=HTTP_201_CREATED)
 
 
+@delete("/api/v1/registrations/{registration_id:int}")
+async def unregister(db_session: AsyncSession, registration_id: int) -> None:
+    registration = await get_registration_by_id(registration_id, db_session)
+    await delete_registration(registration, db_session)
+    return None
+
+
```

Comment:
I believe we should now use Litestar sqlalchemy's services ([link to the `delete` helper](https://docs.advanced-alchemy.litestar.dev/latest/reference/service.html#advanced_alchemy.service.SQLAlchemyAsyncRepositoryService.delete)) instead of implementing our own stuff.

Something like

```suggestion
async def unregister(registration_service: RegistrationService, db_session: AsyncSession, registration_id: int) -> None:
    await registration_service.delete(registration_id)
    return None
```

Reply:
Yes same here, this code was removed after I rebased on main. Thanks for the comment and the code suggestion